debezium-core/src/main/java/io/debezium/relational/history/DatabaseHistory.java

```diff
@@ -48,7 +48,11 @@
     public static final Field DDL_FILTER = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "ddl.filter")
                                                 .withDisplayName("DDL filter")
                                                 .withType(Type.STRING)
-                                                .withDefault("DROP TEMPORARY TABLE IF EXISTS .+ /\\* generated by server \\*/")
+                                                .withDefault(
+                                                    "DROP TEMPORARY TABLE IF EXISTS .+ /\\* generated by server \\*/," +
```

Comment:
Hm, is this assuming this actually is a comma-separated list of reg exps? I'd have expected the patterns to be seprated by "|"?

Reply:
Please check https://github.com/debezium/debezium/blob/68f0a96a08389d6510326754699890b3580884ab/debezium-core/src/main/java/io/debezium/function/Predicates.java#L169 and https://github.com/debezium/debezium/blob/68f0a96a08389d6510326754699890b3580884ab/debezium-core/src/main/java/io/debezium/util/Strings.java#L103
it is the same as for filters and other fields

Reply:
Ah, I see. Will that deal correctly though with reg exps containing "," characters themselves (not to be expected for field names, but it could be the case here)?